jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/model/CodeModernizerSessionContext.kt

```diff
@@ -241,12 +335,21 @@ data class CodeModernizerSessionContext(
                     }
                     return FileVisitResult.CONTINUE
                 }
+
                 override fun visitFileFailed(file: Path?, exc: IOException?): FileVisitResult =
                     FileVisitResult.CONTINUE
             }
         )
         return dependencyfiles
     }
+
+    fun showTransformationHub() = runInEdt {
+        val appModernizerBottomWindow = ToolWindowManager.getInstance(project).getToolWindow(CodeModernizerBottomToolWindowFactory.id)
+            ?: error(message("codemodernizer.toolwindow.problems_window_not_found"))
+        appModernizerBottomWindow.show()
+        CodeModernizerBottomWindowPanelManager.getInstance(project).setJobStartingUI()
+    }
+
     companion object {
```

Comment:
Change the bottom of the IDE to the transformation hub once the IDE bundled maven finished.